governance/polls/Adjust the WBTC Debt Ceiling - July 27, 2020

```diff
@@ -0,0 +1,35 @@
+---
+title: Adjust the WBTC Debt Ceiling - July 27, 2020
+summary: Signal your support for increasing the WBTC Debt Ceiling from 20MM to 34MM.
+discussion_link: https://forum.makerdao.com/t/signal-request-should-we-raise-wbtc-debt-ceiling/3223?u=andy_mccall
```

Comment:
I don't think it matters, but maybe strip out the 'u=andy_mccall' and the topic title from the link?

Reply:
(Leaving just the topic ID)

Reply:
👍